bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/blindHandler.java

```diff
@@ -0,0 +1,190 @@
+            case DOWN:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousDown(id);
+                }
+                break;
+            case SHUTTER:
+                if (command.toFullString() == DOWN) {
+                    dominoswissHandler.continuousDown(id);
+                } else if (command.toFullString() == UP) {
+                    dominoswissHandler.continuousUp(id);
+                } else if (command.toFullString() == CHANNEL_STOP) {
+                    dominoswissHandler.stop(id);
+                } else {
+                    logger.debug("Blind got command but nothing executed: {}  and ChannelUID: {}",
+                            command.toFullString(), channelUID.getIdWithoutGroup());
+                }
+
+            case TILTDOWN:
+                if (command instanceof Number) {
+                    dominoswissHandler.tiltDown(id);
+                }
+                break;
+
+            case TILTUP:
+                if (command instanceof Number) {
+                    dominoswissHandler.tiltUp(id);
+                }
+                break;
+
+            case TILT:
+                if (command.toFullString() == UP) {
+                    dominoswissHandler.pulseUp(id);
+                } else if (command.toFullString() == DOWN) {
+                    dominoswissHandler.pulseDown(id);
+                } else if (command.toFullString() == CHANNEL_STOP) {
+                    dominoswissHandler.stop(id);
+                } else {
+                    logger.debug("Blind got command but nothing executed: {}  and ChannelUID: {}",
+                            command.toFullString(), channelUID.getIdWithoutGroup());
+                }
+
+            default:
+                break;
+        }
+    }
+
+    @SuppressWarnings("null")
+    @Override
+    public void initialize() {
+        this.id = getConfig().as(blindConfig.class).id;
+        updateBridgeStatus();
+        dominoswissHandler.registerBlind(this.id, getThing().getUID());
+    }
+
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        updateBridgeStatus();
+    }
+
+    /*
+     * Gets the ID of this Blind
+     */
+    public String getID() {
+        return this.id;
+    }
+
+    @SuppressWarnings("null")
+    private void updateBridgeStatus() {
+        try {
+            ThingStatus bridgeStatus = getBridge().getStatus();
+            if (bridgeStatus == ThingStatus.ONLINE && getThing().getStatus() != ThingStatus.ONLINE) {
+                updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE);
+                dominoswissHandler = (eGateHandler) getBridge().getHandler();
+            } else if (bridgeStatus == ThingStatus.OFFLINE) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+            }
+        } catch (Exception e) {
+            logger.error("Could not update ThingStatus ", e);
+
+        }
+    }
```

Comment:
What is the purpose of this? Actually the framework takes care that the bridge's child Things statuses are set correctly dependent on the bridge status.

Reply:
Removed thanks for the hint